### PR TITLE
adding ceaser cipher using python

### LINKED ISSUE #17: Caesar Cipher
# Description

Implement a Caesar cipher, both encoding and decoding. The key is an integer from -25 to 25. This cipher rotates the letters of the alphabet (a-z/A-Z). The script should have both functions `encode` and `decode` and the output should first encode the input message according to the key provided in the input and then decode it to demonstrate both the functions.

### Example:
```
If the message is: "Hi, How are you?" and key is: -17

encode(message) should output: "Qr, Qxf jan hxd?"

and decode(secret_message) should output: "Hi, How are you?" for secret_message="Qr, Qxf jan hxd?"
```

## Language

- [x] C++: @ShivamPorwal02 
- [ ] Java: @
- [ ] Python: @
- [ ] C: @
- [ ] Other (Please specify)

## Type of issue

- [x] Feature
- [ ] Bug

The checklist will be updated by the maintainers / author of this issue according to the language in which snippets are added or are proposed to be added.

### PR DESCRIPTION
**Pull request by aolaria**
# Ceaser Cipher

Fixes: #17 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generates no new warnings

### CODE REVIEW

File: snippets/python/ceaser_cipher.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+## Description
+This script uses the [Ceaser Cipher](https://en.wikipedia.org/wiki/Caesar_cipher) for encrypting messages in plain text.
+
+
+### Options: 
+    -e: encrypts plain text
+    -d: decodes encrypted text
+    -k, --key
+
+## Example:
```
Comment (sakshamarora1) [inline snippets/python/ceaser_cipher.md]: We should move this below the `Code`

File: snippets/python/ceaser_cipher.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
+# Ceaser Cipher - Python3
```
Comment (sakshamarora1) [inline snippets/python/ceaser_cipher.md]:
```suggestion
# Caesar Cipher
```

Comment (sakshamarora1) [reply snippets/python/ceaser_cipher.md]: Looks like the spelling needs to be updated everywhere :)

**Review by sakshamarora1 — CHANGES_REQUESTED** [review]
Really great work @aolaria! Just a few stuff that needs updating.

Also please update the PR description accordingly.

File: snippets/python/ceaser_cipher.md (hunk 2 above)
Comment (aolaria) [reply snippets/python/ceaser_cipher.md]: Done, I think I've already fixed all. let me know if there's something more :)

File: snippets/python/ceaser_cipher.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+### Options: 
+    -e: encrypts plain text
+    -d: decodes encrypted text
+    -k, --key
+
+
+## Code:
+```python
+#!/usr/bin/python3
```
Comment (kaustubhgupta) [inline snippets/python/ceaser_cipher.md]: Remove these comments

File: snippets/python/ceaser_cipher.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
+# Ceaser Cipher
+
+## Description
+This script uses the [Ceaser Cipher](https://en.wikipedia.org/wiki/Caesar_cipher) for encrypting messages in plain text.
+
+
+### Options:
```
Comment (kaustubhgupta) [inline snippets/python/ceaser_cipher.md]: This needs to be level 2 (##)

File: snippets/python/ceaser_cipher.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
+# Ceaser Cipher
+
+## Description
+This script uses the [Ceaser Cipher](https://en.wikipedia.org/wiki/Caesar_cipher) for encrypting messages in plain text.
+
+
+### Options: 
+    -e: encrypts plain text
+    -d: decodes encrypted text
+    -k, --key
```
Comment (kaustubhgupta) [inline snippets/python/ceaser_cipher.md]: Shouldn't this be:
-k : key ?

File: snippets/python/ceaser_cipher.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+    return result
+
+
+if __name__ == "__main__":
+    options = ''' 
+    usage: 
+
+    -e: encrypts plain text
+    -d: decodes encrypted text
+    -k, --key
```
Comment (kaustubhgupta) [inline snippets/python/ceaser_cipher.md]: Same thing here

File: snippets/python/ceaser_cipher.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+        d_msg = decode(msg, key)
+        print(f"decoded message: {d_msg}")
+        sys.exit(0)
+    else:
+        print(f"error: you must specify one option\n\n:{options}")
+        sys.exit(1)
+```
+
+
+## Example:
```
Comment (kaustubhgupta) [inline snippets/python/ceaser_cipher.md]: Instead of example, follow our pattern of Input and Output

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: snippets/python/ceaser_cipher.md (hunk 7 above)
Comment (aolaria) [reply snippets/python/ceaser_cipher.md]: don't know what you mean with this.

Comment (aolaria) [reply snippets/python/ceaser_cipher.md]: if you give me some feedback I'll fix it right away

Comment (aolaria) [reply snippets/python/ceaser_cipher.md]: I think might be ok right now

Comment (aolaria) [reply snippets/python/ceaser_cipher.md]: @kaustubhgupta Hey! good afternoon!, could you please check this PR?

File: snippets/python/ceaser_cipher.md (hunk 2 above)
Comment (sakshamarora1) [reply snippets/python/ceaser_cipher.md]: Looks like you misspelled for `Caesar` everywhere. It's Caesar (and not Ceaser) :)

Comment (aolaria) [reply snippets/python/ceaser_cipher.md]: oh Je! you're actually right and I didn't notice it hahaha really sorry. it's done.

**Review by kaustubhgupta — APPROVED** [review]

**Pull request merged**